apps/desktop/src/modules/shared/env-utils.spec.ts

```
@@ -0,0 +1,275 @@
+      expect(ENV_ALLOWLIST).toContain('CARGO_HOME');
+    });
+  });
+
+  // ================================================================
+  // ENV_BLOCKLIST_PATTERNS
+  // ================================================================
+  describe('ENV_BLOCKLIST_PATTERNS', () => {
+    it('should be a non-empty array of RegExp', () => {
+      expect(Array.isArray(ENV_BLOCKLIST_PATTERNS)).toBe(true);
+      expect(ENV_BLOCKLIST_PATTERNS.length).toBeGreaterThan(0);
+      ENV_BLOCKLIST_PATTERNS.forEach(p => expect(p).toBeInstanceOf(RegExp));
```

Biome (2.3.14)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)